src/UIComponents/ModelCard.jsx

```diff
@@ -31,6 +31,8 @@ class ModelCard extends Component {
       datasetDetails
     } = this.props;
     console.log("trainedModelDetails", trainedModelDetails)
+    const predictionStatement = I18n.t("predictionStatement",
+      {"output": label.id, "inputs": selectedFeatures.join(", ")})
```

Comment:
Do we plan to eventually update this to not make the assumption of comma delimited lists as well?

Reply:
No plan right now because we don't have a good solution. We would need some library which is an expert on localizing lists of items. Maybe a simple solution would be building a library of our own locale specific "comma" characters. Right now, product is aware of the issue and comfortable moving forward with the english comma.